docs/source/guide/ses-example-verify.rst

```diff
@@ -1,128 +0,0 @@
-.. Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Doesn't deleting this break the link to:
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/ses-example-verify.html

Reply:
No. That file was never referenced in the docs. Instead, the file ses-verify.rst is used. I don't know what the history of the file is so I don't know why the duplicates exist.

Where is there a link to the ses-example-verify.html file? The docs link to ses-verify.html.

Reply:
I agree that the odds a link to this page are unlikely due to the lack of discoverability since it was never linked to on another page of the documentation. However, a direct link like the above will generate a 404 if we merge this.